doc/todo/RELEASE

```diff
@@ -25,265 +12,78 @@ Following shell expansions are used in this document:
+
+= When Source Code is considered ready =
+
+- Trigger release pipeline with parameters
+  On a release of a new version, the revision parameters should be set to 1 (default)
+- Download artifacts after release build is finished and pipeline waits for input:
+  To download the artifacts either open the "Artifacts" tab on Blue Ocean or open this URL
+  https://build.libelektra.org/job/libelektra-release/<BUILD_NUMBER>/artifact/artifacts/
+  where <BUILD_NUMBER> is the number of the Jenkins build e.g. 77
+- Inspect the logs from the artifacts
+  For each distribution two artifact folder exist: "<DISTRO_CODENAME>" and "<DISTRO_CODENAME>-installed"
+  "<DISTRO_CODENAME>-installed" contains the test and strace logs of Elektra installed through the built packages
+  For <DISTRO_CODENAME> artifact:
+    - [ ] Error logs (*.error) must be empty
+    - [ ] Check in the (test-*) folders if the passed tests really were successful
+    - [ ] Check if version is correct in the version file
+    - [ ] Check if the package build log has warnings (<DISTRO_CODENAME>/elektra_*.build and <DISTRO_CODENAME>/elektra_*.error)
+    - [ ] Check if all plugins/bindings/tools that are part of a package were included (<DISTRO_CODENAME>/elektra_*.build)
+  For <DISTRO_CODENAME>-installed artifact:
+    - [ ] Check in the (test-*) folders if the passed tests really were successful
+- Inspect the changes to the libelektra git repository in git/master.log
+  - [ ] Check diff of commit "Regenerate dot of plugins for release" and if
+        doc/images/plugins.pdf was generated correctly
+  - [ ] Check diff of commit "Update plugin info status for release": Plugin tags should be sorted (optional commit)
+  - [ ] Check diff of commit "Update debian/changelog for release": Changelog should have a new entry for this release
+  - [ ] Check diff of commit "Update fedora/changelog for release": Changelog should have a new entry for this release
 
 = Publish =
 
-== Debian Package ==
-
- cd ~elektra/$VERSION/debian/$DVERSION && reprepro --ignore=wrongdistribution include stretch elektra_${DVERSION}_`dpkg-architecture -qDEB_BUILD_ARCH`.changes && update-reprepro
-
-== Commit source package ==
+- Publish packages and artifacts from the pipeline:
+  - Go to the classic Jenkins UI (BlueOcean sometimes doesn't show the Input step on long pipelines)
+  - Select "Paused for Input" on the side menu
+  - On the prompt "Publish Release?" select "Yes" to publish the artifacts
+- Verify that debs.libelektra.org and rpms.libelektra.org contain the released packages.
+- Clean the git statistics in git/statistics and add them to the release notes (formatting, remove uninteresting data)
+- Add hash sums to release notes and run linkchecker to verify that all download links of the current release are working:
+  mkdir build && cd build && make html && ../scripts/link-checker external-links.txt
+- (Possible) rebuilds from website for last changes in the release notes
```

Comment:
At least one rebuild is necessary. But it happens anyway, so isn't this step simply "waiting for master build to finish and verify website is correct".